client/components/popover/menu.jsx

```diff
@@ -34,21 +38,29 @@ const PopoverMenu = React.createClass( {
 
 	render: function() {
 		const children = React.Children.map( this.props.children, this._setPropsOnChild, this );
-
+		const PopoverComponent = this.props.popoverComponent;
```

Comment:
Allows a `Popover` component to be specified, so we can hand it `ReaderPopover`.